app/javascript/dashboard/api/auth.js

```diff
@@ -142,9 +142,9 @@ export default {
   }) {
     const formData = new FormData();
     Object.keys(profileAttributes).forEach(key => {
-      const value = profileAttributes[key];
-      if (value) {
-        formData.append(`profile[${key}]`, value);
+      const hasValue = profileAttributes[key] === undefined;
```

Comment:
Why is there an explicit check? Can't we do !profileAttributes[key]?

Reply:
Boolean values like `false` will be ignored if we don't do explicit check